float32 x_max
float32 x_min
float32 y_max
float32 y_min
---
float32 x
float32 y
float32 theta
